src/schema_factory.spec.ts

```diff
@@ -1,6 +1,7 @@
 import 'reflect-metadata';
 
 import * as D from './decorator';
+import * as DD from './decorator/';
```

Comment:
I fell a little bit uneasy with these "duplicated" decorators

Reply:
managed to expose everything from inside the old `decorator.ts` file, guess it is better like that while all the other decorators are in the old structure.